toolchain/check/import_ref.cpp

```diff
@@ -1844,24 +1846,26 @@ class ImportRefResolver {
   auto FinishPendingGeneric(PendingGeneric pending) -> void {
     const auto& import_generic = import_ir_.generics().Get(pending.import_id);
 
-    // Note that the generics list can be reallocated by ResolveLocalEvalBlock
-    // importing more generics.
+    // Don't store this between calls: the generics list can be reallocated by
+    // ResolveLocalEvalBlock importing more generics.
+    auto local_generic = [&]() -> auto& {
+      return context_.generics().Get(pending.local_id);
+    };
```

Comment:
Maybe move onto the type as a function, so calls look like `pending.Get(context)`? (lambdas always make me hesitant)

Reply:
Not, discussed offline, just inlining the fetches